### PR TITLE
Add Project reference for UWP to GoogleMaps.UWP

### PR DESCRIPTION
**Pull request by JKennedy24**
Added Project Reference for UWP sample to GoogleMaps.UWP. Also changed configuration settings for Android and UWP to deploy

### CODE REVIEW

File: XFGoogleMapSample/Droid/XFGoogleMapSample.Droid.local.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>XFGoogleMapSample.Droid</RootNamespace>
     <AssemblyName>XFGoogleMapSample.Droid</AssemblyName>
-    <TargetFrameworkVersion>v6.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v7.0</TargetFrameworkVersion>
```
Comment (deextercz) [inline XFGoogleMapSample/Droid/XFGoogleMapSample.Droid.local.csproj]: if @amay077 will agree with version 7.0 than I will approve

**Comment by amay077** [comment]
You should not contains Android project file changes.
Because this PR's context is "UWP" project.

Changes should be "atomic" as also @atsushieno said.

**Comment by JKennedy24** [comment]
I will undo the android project file changes and commit

**Comment by JKennedy24** [comment]
To be honest, I still don't really know what I'm doing. All the code is there. but Github is so difficult to use compared to TFS

**Comment by deextercz** [comment]
Try to use git extensions as git client

**Comment by amay077** [comment]
I'm using SourceTree which is best git client for Mac/Windows.

I'll close this PR.
I think you had better retry fork and create branch.

Don't worry, you can try again and again. 👍

**Comment by JKennedy24** [comment]
My Project wont let me undo those changes to the Android project. Every time I undo it just comes straight back. It must be something on my computer changing it from 6 to 7

**Comment by amay077** [comment]
If you use Visual Studio git extension or TFS then you had better disable it.
Because I think VS or TFS git extensions are not for beginners.

Alternatively, you can use more better git clients listed bellows:

* [GitHub Desktop - Simple collaboration from your desktop](https://desktop.github.com/)
* [SourceTree - A free Git & Mercurial client for Windows or Mac | Atlassian](https://www.atlassian.com/software/sourcetree)
* Or git command line interface

First, You should be careful to which files to commit. You should make only one commit per one reason.

Visual Studio often changes to project/solution and some auto generated files automaticaly.
But these are unrelated for your purpose.

So you should not contains these files to your commit.

Next, You should be careful to not contains unintended file changes before push operation. Push operation can not undo.

**Comment by deextercz** [comment]
If you dont know how to do it create pull request to my branche https://github.com/deextercz/Xamarin.Forms.GoogleMaps/tree/to_merge and I will modify this and create new one to amay0777 repository

**Comment by JKennedy24** [comment]
@deextercz Have submitted a pull request to your branch. If that works, I will do a similar thing with the rest of my pull requests. Sorry for making such a mess of it

**Comment by JKennedy24** [comment]
I am using Visual Studio Git extensions. I think perhaps the functionality is very limited. This may be why it is causing so much pain in excluding files from commits.

**Pull request closed without merging**